### ISSUE TITLE
B for black mode

### ISSUE THREAD

**Issue by pepelsbey** [shower/shower#97]
Something like **B** (black) or **W** (white) modes in PowerPoint or Keynote. There's no special need in W mode, but B could be useful. B is actually toggling between black screen and current slide, but not really toggling… it's complicated :)

So this feature request contains of three points:
- It's should be possible to go to `?full` mode without switching to the first slide.
- B should switch to `?full` mode only from `?full#ID` mode, not from the list mode.
- Every key on a keyboard switches presntation back to previous or to the first slide in `?full#ID` mode

**Comment by shvaikalesh** [comment]
We can utilize `prefers-color-scheme` media query and `AmbientLightSensor` to detect wanted mode. If we had "presenter view", we would definitely know that dark mode is undesired.

**Comment by pepelsbey** [comment]
No-no, black mode is just a black slide. Typical thing for PowerPoint and Keynote. Some clickers even have a dedicated B button.
